funding/manager.go

```diff
@@ -3159,6 +3164,28 @@ func (f *Manager) addToRouterGraph(completeChan *channeldb.OpenChannel,
 		return ErrFundingManagerShuttingDown
 	}
 
+	// The user can define non-default channel policies when opening a
+	// channel. They are stored in the database to be persisted from the
+	// moment of funding the channel to it being confirmed. We just
+	// announced those policies to the network, but we also need to update
+	// our local policy in the switch to make sure we can forward payments
+	// with the correct fees. We can't do this when creating the link
+	// initially as that only takes the static channel parameters.
```

Comment:
can we not change the link creation so that it takes these non-static params too? Just so that we never have to worry about the link being started with the incorrect forwarding policy